tests/integrationv2/README.md

```diff
@@ -8,6 +8,7 @@ have all the dependencies installed correctly. The integration test dependencies
  * openssl (based on the S2N_LIBCRYPTO env var)
  * tox
  * Compiled Java SSLSocketClient for the Java provider
+ * Compiled an s2nc executable named s2nc_head in the bin directory for the cross compatibility test
```

Comment:
does this happen automatically or is there manual steps involved for local testing?

Reply:
hmm although the other items dont have instructions either :(

Reply:
Nope, none of these dependencies build automatically. I think this is supposed to be a list of dependencies for people who want to run the integ tests locally. And yes, it should be automated, however we don't have that set up yet.